plugins/MessengerRedisBundle/MessengerRedisBundle.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * @copyright   2022 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+declare(strict_types=1);
+
+namespace MauticPlugin\MessengerRedisBundle;
+
+use Mautic\IntegrationsBundle\Bundle\AbstractPluginBundle;
+
+/**
+ * Class MessengerRedisBundle.
+ */
+class MessengerRedisBundle extends AbstractPluginBundle
```

Comment:
This bundle should be also an external plugin, right?

Reply:
I removed this bundle from this PR, I will create another PR for this specific bundle, so we can break the review process into smaller chunks